### DISCUSSION TITLE
[AXIS P3818-PVE] - video record triggering

### DISCUSSION THREAD

**Discussion by soufianta** [AxisCommunications/discussions#137] [VAPIX] [answered]
Hi everybody,

I need to trigger a video recording on a AXIS panoramic camera (P3818-PVE). I know that this camera can do video recording but I wanted to know, how can I trigger that (through an API) from an external source (server, other equipment etc.)?

Thanks

**Comment by soufianta** [comment]
Hi,

According to below documentation, I don't see anywhere how to trigger that video recording: 
https://www.axis.com/vapix-library/subjects/t10175981/section/t10176034/display

Thanks for the help.

**Comment by mattias-kindborg-at-work** [comment]
When you say *trigger a recording*, what do you mean? The camera can provide you with a video source, but where do you expect the recording to be saved, are you using a SD card? Please elaborate.

**Reply by soufianta** [reply]
Hi Mattias,

Sorry for the late reply, I know that the camera (AXIS P3384-VE) is for the moment setup to record every 5minutes but I would like to see if I can trigger a recording based on a specific 'event' (via the API). I saw that the we can also configure events in the GUI of the camera and I also wanted to send that record to another application (via http eventually). Thanks and a Happy New Year.

**Reply by soufianta** [reply]
Hi @mattias-kindborg-at-work 
The camera is the "P3807-PVE" and I would like to trigger the video recording first (1st step). Then, I would have to know from where I can download it (where is it stored on the camera) and all those things through API calls obviously.

Thanks.

**Reply by soufianta** [reply]
@mattias-kindborg-at-work 
On the GUI of the camera, I can "record" but I first need to configure a storage (sd-card). Then I'll trigger the record button and see which request is send. Once I've located the recording, I'll have to send it through HTTP/HTTPS to back-office server that will show it somewhere (final step).

**Reply by mattias-kindborg-at-work** [reply, marked as answer]
We have a sample showcasing how one would be able to [send images to Azure](https://github.com/AxisCommunications/acap-integration-examples-azure/tree/main/images-to-azure-storage-account). Please look at the sample and investigate whether it could be adapted to send video to your back-office service instead.
